### ISSUE TITLE
Soundboard: Add record sound feature

### ISSUE THREAD

**Issue by jonas747** [jonas747/yagpdb#96]
Add the ability to directly record a soundboard sound from a discord channel.

How to accomplish this:
 1. Record and decode each indiv users audio into pcm with timestamps 
 2. Mix togheter the recorded pcm frames (adding + clipping should be enough?), if there is no audio for a given user in a frame, assume silence
 3. Add it to the soundboard transcode queue

**Comment by davidjameshowell** [comment]
Would this be triggered by a command to begin recording the specific soundbyte intended?

**Comment by jonas747** [comment]
yeah it wont be constantly recording, you would invoke a command like `-record` and `-stop`

maybe ill add another command to also trim it.
